### PR TITLE
Installation option refactoring to be more concise

### PR DESCRIPTION
**Pull request by leanderweber**
### Description
- Refactored installation options. E.g., "extras" is removed, instead singular XAI libraries are targetable.

### Implemented changes
  - [x] Refactor installation options (single XAI libraries, also options "test", "full")
  - [x] update docs

### Minimum acceptance criteria
- [x] all tests run
- [x] Reviewed by @annahedstroem

### CODE REVIEW

File: docs/source/getting_started/installation.md [hunk 1]
```diff
@@ -41,15 +41,15 @@ The following XAI Libraries are currently supported:
 To enable the use of wrappers around [Captum](https://captum.ai/), you need to have PyTorch already installed and can then run
 
 ```setup
-pip install quantus[extras]
+pip install quantus[captum]
```
Comment (annahedstroem) [inline docs/source/getting_started/installation.md:44]: nice!

File: docs/source/getting_started/installation.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ### Installing from PyPI
 
 If you already have [PyTorch](https://pytorch.org/) or [Tensorflow](https://www.tensorflow.org) installed on your machine, 
-Quantus can be obtained from [PyPI](https://pypi.org/project/quantus/) as follows:
+the most light-weight version of Quantus can be obtained from [PyPI](https://pypi.org/project/quantus/) as follows:
```
Comment (annahedstroem) [inline docs/source/getting_started/installation.md:6]: maybe add something like
"This means that additional explainability functionality will not be included."

**Comment by annahedstroem** [comment]
See separate comments on why separating this PR and this one: https://github.com/understandable-machine-intelligence-lab/Quantus/pull/166

File: docs/source/getting_started/installation.md (hunk 2 above)
Comment (leanderweber) [reply docs/source/getting_started/installation.md]: added.

**Pull request merged**